dist/angular-promise-buttons.js

```diff
@@ -2,7 +2,7 @@ angular.module('angularPromiseButtons', [
 ]);
 
```

Comment:
Don't make the changes here. They need to be inside the `src` folder and use `grunt serve` for development. Otherwise they'll be overwritten on the next update.

Also: The existing unit tests must not fail and it wouldn't hurt to add new ones for the new functionality ;)